### PR TITLE
added channel play file

### PR DESCRIPTION
**Pull request by thjunge11**
Hi Patrick

the play URI channel did not work with local files due to the service = webradio setting, so i created a new channel to play files by giving an absolute path string as uri to the channel. It is tested and works, so feel free to merge to 0.0.11 :)

### CODE REVIEW

File: ESH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -119,5 +120,11 @@
         <label>Play URI</label>
         <description>Play the stream at given uri</description>
     </channel-type>
+    
+    <channel-type id="playFile" advanced="true">
+        <item-type>String</item-type>
+        <label>Play file</label>
+        <description>Play a file, located on your Volumio2 device at the given absolute path, e.g. "mnt/INTERNAL/song.mp3"</description>
```
Comment (patrickse) [inline ESH-INF/thing/thing-types.xml:127]: There´s a little typo within the example for the absolute path... :) the path is not absoulte... I guess it should look like this...
```
/mnt/INTERNAL/song.mp3
```

Comment (patrickse) [reply ESH-INF/thing/thing-types.xml:127]: @thjunge11 Many thanks for you pull request... 👍

Comment (thjunge11) [reply ESH-INF/thing/thing-types.xml:127]: @patrickse: The path needs to be absolute but without leading "/" for the uri in the json.

Volumio does find:  
`2017-10-04T16:31:02.447Z - info: ADDING THIS ITEM TO QUEUE: {"service":"mpd","title":"","uri":"mnt/INTERNAL/sound.wav"}`
but would not find:
`2017-10-04T16:31:02.447Z - info: ADDING THIS ITEM TO QUEUE: {"service":"mpd","title":"","uri":"/mnt/INTERNAL/sound.wav"}`

do not know why, to be honest?

Comment (patrickse) [reply ESH-INF/thing/thing-types.xml:127]: Oh ok.. got it... so anyway thanks for the cool addition... ;)

**Pull request merged**